sale_comment_template/tests/test_sale_order_report.py

```diff
@@ -1,41 +1,47 @@
 # Copyright 2017 Simone Rubino - Agile Business Group
 # Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 import odoo.tests
 from odoo.tests.common import TransactionCase
 
 
 @odoo.tests.common.at_install(False)
```

Comment:
I think you also need to remove this line, otherwise the tests will be completely skipped. I am not sure, though, have you tested this and made sure the tests actually ran locally?
```suggestion
```